SQLAlchemy_db_connector/repositories/orders_repository.py

```diff
@@ -0,0 +1,38 @@
+from SQLAlchemy_db_connector import connection_db
+from SQLAlchemy_db_connector.models.orders import Orders
+from SQLAlchemy_db_connector.models.products import Products
+from sqlalchemy import select, delete, update
+
+
+class OrdersRepository:
+    def __init__(self):
+        self.__session = connection_db.session
+        self.__result_list = []
+
+    def get_all_orders(self) -> list:
+        all_orders = self.__session.query(Orders).all()
+        return all_orders
+
+    def get_by_id(self, id_value: int):
```

Comment:
if I remember correctly all pony methods required decorator

Reply:
I don't know, all work correctly

Reply:
Sorry, my fault. I thought it is PonyOrm implementation. Comments are not relevant